doc/tutorials/SIR-X.ipynb

```diff
@@ -323,14 +326,18 @@
    "outputs": [],
    "source": [
     "# Calculate confidence intervals\n",
-    "mse_avg, mse_list, p_list = g_sirx.ci_block_cv(lags = 1, min_sample=3)"
+    "mse_avg, mse_list, p_list, pred_data = g_sirx.block_cv(lags = 1, min_sample=6)"
```

Comment:
We should be consistent with the use of spaces between "=" signs. `lags = 1, min_sample=6` should probably be `lags = 1, min_sample = 6`

pylint is not running in .ipynb files atm

Reply:
Thats true. Now I corrected it